### PR TITLE
Feature/demo libraray page

### PR DESCRIPTION
**Pull request by danial-rokt**
Add demo library page
Simulator Screen Shot - iPhone 12 - 2021-04-12 at 15 17 31

### CODE REVIEW

File: RoktDemo/Views/SubViews/DemoItemView.swift [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        VStack(alignment: .leading, spacing: 10) {
+            HStack{
+                if demoItemModel.iconURL.isUrl() {
+                    KFImage.url(
+                        URL(string: demoItemModel.iconURL)
+                    )
+                    .resizable()
+                    .aspectRatio(contentMode: .fit)
+                    .frame(width: 100, height: 60, alignment: .leading)
+                } else if !demoItemModel.iconURL.isEmpty {
```
Comment (nedarokt) [inline RoktDemo/Views/SubViews/DemoItemView.swift:31]: what happens here if demoItemModel.iconURL is not empty, but invalid?

File: RoktDemo/Model/PreDefinedScreen1Model.swift [hunk 2]
```diff
@@ -0,0 +1,18 @@
+//
```
Comment (nedarokt) [inline RoktDemo/Model/PreDefinedScreen1Model.swift:1]: Is there a difference between PredefinedScreenModel 1,2 and 3? Can they all be the same class?

File: RoktDemo/Views/SubViews/DemoItemView.swift (hunk 1 above)
Comment (danial-rokt) [reply RoktDemo/Views/SubViews/DemoItemView.swift:31]: I have tested it. It simply doesn't load the image. I think it is fine for this stage as we don't have any placeholder image to cover the failure images. What do you think?

Comment (nedarokt) [reply RoktDemo/Views/SubViews/DemoItemView.swift:31]: No worries, just making sure it didn't crash

**Review by nedarokt — APPROVED** [review]

**Pull request merged**